include/oneapi/dpl/internal/serial_numeric_impl.h

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (c) Intel Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#ifndef _ONEDPL_SERIAL_NUMERIC_IMPL_H
+#define _ONEDPL_SERIAL_NUMERIC_IMPL_H
+#include <iterator>
+#include <numeric>
```

Comment:
Do we need the `<numeric>` header for `#if __cplusplus < 201703L`? Can it be moved to `#else` branch?